docs/ADR-65-avatar-system-for-renderer.md

```diff
@@ -0,0 +1,370 @@
+# ADR 65 Avatar System for Renderer (Unity)
+
+## Problem Statement
+
+Avatars are a key part of Decentraland, the system behind the curtains must be resilient, scalable and performant.
```

Comment:
Could you add more details about what people is about to read? Does this document solves an issue? a design problem? a refactor? Although I understand that avatars are key part of DCL, it is not clear what decision is being addressed in this document